### PR TITLE
Some registry stuff

### PR DESCRIPTION
**Pull request by Earthcomputer**
I stopped just before it drove me mad

### CODE REVIEW

**Review by Gegy — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Review by shartte — APPROVED** [review]

File: mappings/net/minecraft/util/registry/MutableRegistry.mapping [hunk 1]
```diff
@@ -6,3 +6,15 @@ CLASS net/minecraft/class_2385 net/minecraft/util/registry/MutableRegistry
...
+	METHOD method_31062 replace (Ljava/util/OptionalInt;Lnet/minecraft/class_5321;Ljava/lang/Object;Lcom/mojang/serialization/Lifecycle;)Ljava/lang/Object;
+		COMMENT If the given key is already present in the registry, replaces the entry associated with the given
+		COMMENT key with the new entry. This method asserts that the raw ID is equal to the value already in
+		COMMENT the registry. The raw ID not being present may lead to buggy behavior.
+		COMMENT
+		COMMENT <p>If the given key is not already present in the registry, adds the entry to the registry. If
+		COMMENT {@code rawId} is present, then this method gives the entry this raw ID. Otherwise, uses the
+		COMMENT next available ID.</p>
+		ARG 1 rawId
+			COMMENT
```
Comment (YanisBft) [inline mappings/net/minecraft/util/registry/MutableRegistry.mapping]: empty comment

File: mappings/net/minecraft/util/registry/RegistryKey.mapping [hunk 2]
```diff
@@ -45,3 +45,7 @@ CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
 	METHOD method_29181 of (Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
 		ARG 0 registry
 		ARG 1 value
+	METHOD method_31163 isOf (Lnet/minecraft/class_5321;)Z
+		COMMENT Returns whether this registry key belongs to the given registry (according to its type, not whether the registry actually contains this key).
+		ARG 1 registry
```
Comment (YanisBft) [inline mappings/net/minecraft/util/registry/RegistryKey.mapping:50]:
```suggestion
		ARG 1 key
```

Comment (Earthcomputer) [reply mappings/net/minecraft/util/registry/RegistryKey.mapping:50]: No, this parameter is specifically the key of the registry this key is supposed to be in, not to be confused with being equal to another key in the same registry.

Comment (YanisBft) [reply mappings/net/minecraft/util/registry/RegistryKey.mapping:50]: Hum okay

**Review by YanisBft — APPROVED** [review]

**Pull request merged**